ambry-network/src/main/java/com.github.ambry.network/NetworkMetrics.java

```diff
@@ -78,6 +79,26 @@
   // the count of renegotiation after initial handshake done
   public final Counter sslRenegotiationCount;
 
+  // NetworkClient metrics
```

Comment:
This is not going to work as is as there can be multiple `NetworkClient`s. On that note, how do metrics work for processor/acceptor at the server. All processors share the same `NetworkMetrics`, but initialize individual `Selector`s. Each of those `Selector` call into `initializeSelectorMetrics()` and it looks like only the last selector gets counted towards the metrics.

Could you review that and let us know if there is a problem there? We may have a similar problem here.